HW9/Arrays.cpp

```diff
@@ -0,0 +1,82 @@
+﻿#include <iostream>
+
+
+void translateArray(int numbers[], int size) {
+    std::cout << "Enter " << size << " numbers: " << std::endl;
+    for (int i = 0; i < size; i++) {
+        std::cin >> numbers[i];
+    }
+    for (int i = 0; i < size; i++) {
+        (numbers[i] > 0) ? numbers[i] *= 2 : numbers[i] = 0;
+    }
+    std::cout << "Array elements are:" << std::endl;
+    for (int i = 0; i < size; i++) {
+        std::cout << numbers[i] << " ";
+    }
+    std::cout << std::endl;
+}
+
+void toUppercase(char str[]) {
+    const int a = 96;
```

Comment:
alphaSymbolStart - alphaSymbolEnd
start - end
alphaStart - alphaEnd
firstSymbol - lastSymbol
or similar but with context not just a - b